CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -55,85 +54,71 @@ Images customization is done via overriding of [GiniImages.xcassets](https://git
-  - With title only
-	  -  Title &#8594; `GiniConfiguration.navigationBarCameraTitleHelpButton`
-      
-##### 2. Camera preview
-- Preview frame color &#8594;  `GiniConfiguration.cameraPreviewFrameColor`
-- Guides color &#8594;  `GiniConfiguration.cameraPreviewCornerGuidesColor`
-- Focus large image &#8594; <span style="color:#009EDF">*cameraFocusLarge*</span> image asset
-- Focus large small &#8594; <span style="color:#009EDF">*cameraFocusSmall*</span> image asset
-- Opaque view style (when tool tip is shown)  &#8594;  `GiniConfiguration.toolTipOpaqueBackgroundStyle`
-
-##### 3. Camera buttons container
-- Background color &#8594;  `GiniConfiguration.cameraButtonsViewBackgroundColor`
-- Container view background color under the home indicator  &#8594;  `GiniConfiguration.cameraContainerViewBackgroundColor` 
-- Capture button
-  - Image &#8594; <span style="color:#009EDF">*cameraCaptureButton*</span> image asset
-- Import button
-	- Image &#8594; <span style="color:#009EDF">*documentImportButton*</span> image asset
-- Captured images stack indicator color &#8594; `GiniConfiguration.imagesStackIndicatorLabelTextcolor`
-- Flash toggle can be enabled through &#8594; `GiniConfiguration.flashToggleEnabled`
-- Flash button
-    - Image &#8594; <span style="color:#009EDF">*flashOn*</span> image asset
-    - Image &#8594; <span style="color:#009EDF">*flashOff*</span> image asset
-
-##### 4. QR code popup
-<br>
-<center><img src="img/Customization guide/QR code popup.jpg" height="70"/></center>
-</br>
-- Background color &#8594;  `GiniConfiguration.qrCodePopupBackgroundColor` using `GiniColor` with dark mode and light mode colors
-- Button color &#8594;  `GiniConfiguration.qrCodePopupButtonColor`
-- Text color &#8594;  `GiniConfiguration.qrCodePopupTextColor` using `GiniColor` with dark mode and light mode colors
-- Title &#8594; <span style="color:#009EDF">*ginicapture.camera.qrCodeDetectedPopup.buttonTitle*</span> localized string
-- Message &#8594; <span style="color:#009EDF">*ginicapture.camera.qrCodeDetectedPopup.message*</span> localized string
+### Single Page
+
+By default, the Gini Capture SDK is configured to capture single page documents.
+No further configuration is required for this.
+
+### Multi-Page
+
+The multi-page feature allows the SDK to capture documents with multiple pages.
+
+To enable this feature simply pass `true` to `GiniConfiguration.shared.multipageEnabled`.
+
+### Camera
+
+* Enable the flash toggle button:
+   To allow users toggle the camera flash pass `true` to `GiniConfiguration.shared.flashToggleEnabled`.
+
+* Turn off flash by default:
+   Flash is on by default, and you can turn it off by passing `false` to `GiniConfiguration.shared.flashOnByDefault`.
+
+ ### QR Code Scanning
+
+When a supported QR code is detected with valid payment data, the QR Code will be processed automatically without any further user interaction.
+The QR Code scanning maybe triggered directly without the need to analyze the document.
```

Comment:
[Suggestion] I think `maybe` should be `may be` here.

Reply:
Good catch!